OpenOversight/app/templates/list_officer.html

```diff
@@ -91,11 +91,9 @@ <h3 class="panel-title accordion-toggle">Officer Rank, Unit, and Status</h3>
                                 name="rank"
                                 class="select2-multi-search"
                                 multiple="multiple">
-                          {% for rank_key, rank in choices['rank'] %}
```

Comment:
For awareness, a future version of wtforms may break this:
https://github.com/pallets-eco/wtforms/issues/811

Reply:
Good call! I will change it so it works for both versions

Reply:
Made that change, it should now also work in the future versions